homeassistant/components/fitbit/sensor.py

```diff
@@ -265,11 +265,12 @@ class FitbitSensorEntityDescription(SensorEntityDescription):
     ),
     FitbitSensorEntityDescription(
         key="activities/steps",
-        name="Steps",
+        translation_key="steps",
         native_unit_of_measurement="steps",
         icon="mdi:walk",
         scope=FitbitScope.ACTIVITY,
         state_class=SensorStateClass.TOTAL_INCREASING,
+        has_entity_name=True,
```

Comment:
I think for Fitbit we should either migrate to has entity name completely (in a separate PR) and not just do only 2 entities

Reply:
Oh I definitely didn't mean to block this PR, but I mostly just expressed my thoughts that Fitbit might not be a good fit for the initial PR. (But maybe I should've been more clear about that, but I wasn't completely sure if I had enough reasons to have that opinion)

Reply:
I've reverted the changes on Fitbit.